connector_search_engine/models/se_index_config.py

```diff
@@ -4,6 +4,8 @@
 
 from odoo import api, fields, models
 
+from odoo.addons.base_sparse_field.models.fields import Serialized
```

Comment:
just to understand why you change this?
it's just because you think it's cleanner and more explicit ? or there is some technical reason ?

Reply:
check the full commit msg :wink: 
Anyway I've done this already in v13 back in the days.

Reply:
I have found the message : https://github.com/OCA/search-engine/commit/5bf7f8536230b72675c1b0558f91b65205af6f6f
Ok so it's a good reason !